src/scrabble.js

```diff
@@ -1,5 +1,35 @@
-function scrabble() {
-  // write code here
+function scrabble(word) {
+  if (!word) return 0;
+  let alphabet = {
+ 
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
```

Comment:
Really good idea to group the characters by value -- SO much better readable for humans! I'm 100% gonna nick that